packages/synapse-react-client/src/style/components/_global-alert.scss

```diff
@@ -1,144 +1,6 @@
-@use '../abstracts/variables' as SRC;
-@use 'sass:map';
-
-@mixin border-radius($radius) {
-  -webkit-border-radius: $radius;
-  -moz-border-radius: $radius;
-  -ms-border-radius: $radius;
-  border-radius: $radius;
-  -o-border-radius: $radius;
-  -khtml-border-radius: $radius;
-}
-
-.FullWidthAlert[role='alert'] {
-  &.global {
-    bottom: 2%;
-    left: 2%;
-    position: fixed;
-    width: 96%;
-    z-index: 2500;
-    filter: drop-shadow(3px 4px 9px #9e9e9e);
-    @include border-radius(2px);
-  }
-  padding: 14px;
-  z-index: 500;
-
-  // Set up default FullWidthAlert styles for all portals
-  color: #323131;
-  @include border-radius(0px);
-  box-shadow: none;
-  margin: 10px 0px;
-  border: none;
-  .alert-link {
-    color: inherit;
-  }
-  &.alert-info {
-    background: #d9ebf2;
-    border-left: 10px solid map.get(SRC.$colors, 'info');
-    .iconBackground {
-      color: map.get(SRC.$colors, 'info');
-    }
-  }
-  &.alert-danger {
-    background: #f7e0dc;
-    border-left: 10px solid map.get(SRC.$colors, 'error');
-    .iconBackground {
-      color: map.get(SRC.$colors, 'error');
-    }
+.SynapseNavDrawerIsShowing .FullWidthAlertGlobal {
+  width: 100%;
+  .FullWidthAlert[role='alert'] {
+    margin-left: 56px;
```

Comment:
Ensures that FullWidthAlert is styled correctly when SynapseNavDrawerIsShowing -- 

main | feature
:---:|:---:
<img width="1398" alt="main_SWC" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/c76c10a0-37fb-4280-9220-d2150bc5e863"> | <img width="1397" alt="feature_SWC" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/1ef37bc9-6d9c-4ab5-8092-10f4b0fceda3">

Reply:
Surprised you don't have to keep `width: calc(100vw - 56px);` for the FullWidthAlert class.